src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+@import 'variables';
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
```

Comment:
The variable `$star-size` is used here but not defined in this file. Ensure it is defined in the imported 'variables' file or within this file.